src/web/configureFontAwesomeLibrary.ts

```diff
@@ -1,20 +1,20 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
-import { faCircleCheck } from '@fortawesome/free-regular-svg-icons';
 import {
   faCheck,
   faChevronDown,
   faChevronUp,
   faEllipsisH,
   faPencil,
   faTrashCan,
+  faXmark,
 } from '@fortawesome/free-solid-svg-icons';
 
 export function configureFontAwesomeLibrary(): void {
   library.add(faCheck);
-  library.add(faCircleCheck);
```

Comment:
Why remove this? It looks like it's being used in accountPending.tsx.

Reply:
we are using the regular icon rather than the solid icon in accountPending page. I tried add the faCircleCheck from regular lib to the library, but it didn't work. so I am using  `findIconDefinition({ prefix: 'far', iconName: 'circle-check' })` instead